packages/pigment-css-react/README.md

```diff
@@ -854,3 +854,216 @@ function App() {
   )
 }
 ```
+
+## Building a UI library
+
+The key of a UI library is to provide a set of reusable components that can be used and themed across **different projects**.
```

Comment:
I'd probably mention that the approach we'll follow here (using slots) is meant for folks wanting to build a highly customizable component. If that's not the case (i.e., you're good with having the component a bit more closed off), then this could be a bit of overkill.

Reply:
Probably need @samuelsycamore advice here. My intention is to tell developers that you want to read this section if you are building a UI library that will be shared among projects. If you want to build components for just one project, this approach might be over-engineered.